2022-05-20 12:26:27,902 INFO  ==============================================================
2022-05-20 12:26:27,903 INFO  Id:    SoftwareManagement.sm_docker_install_remove_multiple
2022-05-20 12:26:27,904 INFO  Dir:   software_management_end_to_end/
2022-05-20 12:26:27,905 INFO  Title: Validate end to end behaviour for the docker plugin for multiple docker images with mixed versions
2022-05-20 12:26:27,907 INFO  ==============================================================
2022-05-20 12:26:27,917 INFO  Using sw id database: {'asciijump': '5475369', 'robotfindskitten': '5474869', 'squirrel3': '5475279', 'rolldice': '5152439', 'moon-buggy': '5439204', 'apple': '5495053', 'banana': '5494888', 'cherry': '5495382', 'watermelon': '5494510', 'registry': '8018911', 'hello-world': '8021526', 'docker/getting-started': '8021973', 'alpine': '7991792'}
2022-05-20 12:26:27,918 INFO  EnvironmentC8y Setup
2022-05-20 12:26:28,479 INFO  Executed systemctl<serv_mapper1>, exit status 3
2022-05-20 12:26:28,589 INFO  Restarting mosquitto too frequently in the last 10 seconds. It was only up for 5 seconds
2022-05-20 12:26:28,591 INFO  Delaying execution by 6 seconds
2022-05-20 12:26:48,483 INFO  Executed sudo<tedge_connect_c8y>, exit status 0, duration 13 secs
2022-05-20 12:26:49,149 INFO  Executed sudo<tedge_connect_c8y_test>, exit status 0
2022-05-20 12:26:49,613 INFO  Executed systemctl<serv_mosq2>, exit status 0
2022-05-20 12:26:50,077 INFO  Executed systemctl<serv_mapper3>, exit status 0
2022-05-20 12:26:52,325 INFO  State of current operation: SUCCESSFUL
2022-05-20 12:26:52,560 INFO  Response status: 201
2022-05-20 12:26:52,561 INFO  Response to action: {
    "creationTime": "2022-05-20T12:26:52.503Z",
    "deviceId": "19397480",
    "deviceName": "m32sd10d",
    "self": "https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19409544",
    "id": "19409544",
    "status": "PENDING",
    "description": "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]",
    "c8y_SoftwareUpdate": [
        {
            "name": "hello-world",
            "action": "install",
            "id": "8021526",
            "version": "::docker",
            "url": ""
        },
        {
            "name": "registry",
            "action": "install",
            "id": "8018911",
            "version": "2.6.2::docker",
            "url": ""
        }
    ]
}
2022-05-20 12:26:52,562 INFO  Started operation: {'creationTime': '2022-05-20T12:26:52.503Z', 'deviceId': '19397480', 'deviceName': 'm32sd10d', 'self': 'https://t493319102.eu-latest.cumulocity.com/devicecontrol/operations/19409544', 'id': '19409544', 'status': 'PENDING', 'description': "Apply software changes, triggered from PySys: [{'action': 'install', 'id': '8021526', 'name': 'hello-world', 'url': '', 'version': '::docker'}, {'action': 'install', 'id': '8018911', 'name': 'registry', 'url': '', 'version': '2.6.2::docker'}]", 'c8y_SoftwareUpdate': [{'name': 'hello-world', 'action': 'install', 'id': '8021526', 'version': '::docker', 'url': ''}, {'name': 'registry', 'action': 'install', 'id': '8018911', 'version': '2.6.2::docker', 'url': ''}]}
2022-05-20 12:26:54,790 INFO  State of operation 19409544 : PENDING
2022-05-20 12:26:57,007 INFO  State of operation 19409544 : EXECUTING
2022-05-20 12:26:59,242 INFO  State of operation 19409544 : EXECUTING
2022-05-20 12:27:01,483 INFO  State of operation 19409544 : EXECUTING
2022-05-20 12:27:03,699 INFO  State of operation 19409544 : FAILED
2022-05-20 12:27:03,700 ERROR Stopping as the operation has failed
2022-05-20 12:27:03,702 WARN  SystemError occurred while running test - The operation has failed
Traceback (most recent call last):
  File "/home/pi/env-pysys/lib/python3.7/site-packages/pysys/baserunner.py", line 1292, in __call__
    self.testObj.setup()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/sm_docker_install_remove_multiple/run.py", line 55, in setup
    self.wait_until_succcess()
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 324, in wait_until_succcess
    self.wait_until_status("SUCCESSFUL")
  File "/home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/software_management_end_to_end/environment_sm_management.py", line 361, in wait_until_status
    raise SystemError("The operation has failed")
SystemError: The operation has failed
2022-05-20 12:27:03,726 WARN  SystemError: The operation has failed ... blocked 
2022-05-20 12:27:03,727 INFO  
2022-05-20 12:27:03,728 INFO  cleanup:
2022-05-20 12:27:07,615 INFO  Executed sudo<tedge_disconnect_c8y>, exit status 0
2022-05-20 12:27:08,177 INFO  Executed systemctl<serv_mapper5>, exit status 3
2022-05-20 12:27:08,182 INFO  
2022-05-20 12:27:08,182 INFO  Test duration: 40.29 secs
2022-05-20 12:27:08,184 INFO  Test final outcome:  BLOCKED
2022-05-20 12:27:08,184 INFO  Test outcome reason: SystemError: The operation has failed
2022-05-20 12:27:08,185 INFO  
